Changelog.md

```diff
@@ -18,6 +18,7 @@ Bugfixes:
  * Assembly output: Do not mix in/out jump annotations with arguments.
  * Code Generator: Annotate jump from calldata decoder to function as "jump in".
  * Type Checker: Properly detect different return types when overriding an external interface function with a public contract function.
+ * Type Checker: Do not crash when a field of a struct used as a parameter in a function type has a non-existent type.
```

Comment:
I think we should not use the word 'crash' for internal errors. Crashes are null pointer dereference or uncaught exceptions.